### ISSUE TITLE
Propick causes crash upon hitting "F"

### ISSUE THREAD

**Issue by JabZ-TF2** [anegostudios/VintageStory-Issues#481]
```
Running on 64 bit Windows with 8 GB RAM 
Version: v1.12.9-rc.2 (Unstable)20.03.2020 00:08:43: Critical error occurred
System.NullReferenceException: Der Objektverweis wurde nicht auf eine Objektinstanz festgelegt.
   bei Vintagestory.Client.NoObf.GuiDialogToolMode.OnRenderGUI(Single deltaTime)
   bei _s7yrQKy4Vbz2SVKGaDNCPw9tM1K._3au83ORVf3OG61Xa24ajTEB8sPO(Single )
   bei _kcKrOSepVEkNdSScAPVPfaxh5aj._4ygjzkACoBBaWylqRAE6aCI1zdv(EnumRenderStage , Single )
   bei _zHtHb9ee7akG2pQQxhr4D2rkwB._4ygjzkACoBBaWylqRAE6aCI1zdv(EnumRenderStage , Single )
   bei _zHtHb9ee7akG2pQQxhr4D2rkwB._VICJKAxdwGWaBnhcWVNrOnChlutA(Single )
   bei _fSI1BMB65cwKGBNcX98G4kfwB7R._VICJKAxdwGWaBnhcWVNrOnChlutA(Single )
   bei _tpQ1IERN2AMQ4hmdmlyG9BevCQP._NCuuF8mJPJiBaOfNuPzBvIB4fs1A(Single )
   bei _tpQ1IERN2AMQ4hmdmlyG9BevCQP._S14BA2XqynDvxHW2ozcDjnseezO(Single )
   bei Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame(Object sender, FrameEventArgs e)
   bei System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   bei OpenTK.GameWindow.RaiseRenderFrame(Double elapsed, Double& timestamp)
   bei OpenTK.GameWindow.DispatchRenderFrame()
   bei OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second)
   bei _kn7PO0uaevAeW8DtMdrhfUZoBG1._sTDuMc98lBVDhW4dFCrBjtOyF8CA(_RubDl5rVwhx3SBZRyR5HVRvNAUf , String[] )
   bei _gTATEydGikdlqm1ooAGFk9fvmWA._sTDuMc98lBVDhW4dFCrBjtOyF8CA(ThreadStart )
-------------------------------
```

**Comment by Bloodsaw** [comment]
I couldnt reproduce this. Maybe u can provide more information? 
Like if u used any mods, if you're able to reproduce this and stuff.

If you are able to reproduce the crash please upload your log files too (like client-main, client-debug)

**Comment by rhonibaer** [comment]
may be this is caused by german language. i had an other issue with /player [name] gamemode {x} with german locale active. in english locale it works fine.

**Comment by Bloodsaw** [comment]
@rhonibaer  Don't think so, I was playing with german language too. By the way the gamemode command was fixed in 1.12.9 stable.

@JabZ-TF2 Is there anyway you are able to reproduce your crash? I guess I will wait a week for feedback from you and if nothing comes I will close this issue as unresolved. You can still comment here if your issue is showing again.

**Comment by copygirl** [comment]
This should be fixed in 1.12.9.
